specification/compute/resource-manager/readme.md

```diff
@@ -360,19 +360,10 @@ input-file:
   - Microsoft.Compute/ComputeRP/stable/2025-04-01/ComputeRP.json
   - Microsoft.Compute/DiskRP/stable/2025-01-02/DiskRP.json
   - Microsoft.Compute/Skus/stable/2021-07-01/skus.json
-  - Microsoft.Compute/GalleryRP/stable/2024-03-03/GalleryRP.json
+  - Microsoft.Compute/GalleryRP/stable/2025-03-03/GalleryRP.json
```

Comment:
@maririos, @mikekistler: Should PRs be allowed to change swagger versions in an existing tag?  Rather than adding a new tag with the new swagger version?  I'm not sure if our BreakingChanges detects this.

Reply:
Short answer is that this should not be allowed, and if our breaking change checks don't catch it then they should be updated so that they do.